packages/sit-onyx/src/components/OnyxFAB/types.ts

```diff
@@ -11,4 +11,9 @@ export type OnyxFABProps = OnyxFABButtonProps & {
    * If unset, the open state is manged internally.
    */
   open?: Nullable<boolean>;
+
+  /**
+   * The offset from the aligned position.
+   */
+  offset?: { x: string; y: string };
```

Comment:
Imo the CSS variables are simple enough. What do you think about removing the prop and just add the example that applies the CSS variables?